### PR TITLE
Use cg.utils.commands in VogueApi

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds Process from cg.utils.commands to vogue api in cg.apps.vogue.

**How to prepare for test**:

On hasta
cd /home/proj/stage/servers/resources/hasta.scilifelab.se

1. bash update-servers-stage.sh master

2. bash update-cg-stage.sh subprocess_vogue

3. bash update-vogue-stage.sh develop

4.  run: 
usestage
export MONGO_URI=mongodb://localhost:27030
export MONGO_DBNAME=vogue-stage
export FLASK_DEBUG=1
export VOGUE_SECRET_KEY=hej
cg -c /home/proj/stage/servers/config/hasta.scilifelab.se/cg-stage.yaml upload vogue genotype -d 300

**Expected outcome**:
Something like this:
----------------- UPLOAD ----------------------
----------------- TRENDING -----------------------
----------------- APPLICATION TAGS -----------------------
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0010P009.
Running command ['/home/proj/bin/conda/envs/S_vogue/bin/vogue', 'load', 'genotype', '-s', '{"status": null, "sample_created_in_genotype_db": "2016-09-15", "sex": null, "comment": null, "_id": "0010P010"}']
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0010P010.
Running command ['/home/proj/bin/conda/envs/S_vogue/bin/vogue', 'load', 'genotype', '-s', '{"status": null, "sample_created_in_genotype_db": "2016-09-15", "sex": null, "comment": null, "_id": "0011P001"}']
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0011P001.
Running command ['/home/proj/bin/conda/envs/S_vogue/bin/vogue', 'load', 'genotype', '-s', '{"status": null, "sample_created_in_genotype_db": "2016-09-15", "sex": null, "comment": null, "_id": "0011P002"}']
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0011P002.
Running command ['/home/proj/bin/conda/envs/S_vogue/bin/vogue', 'load', 'genotype', '-s', '{"status": null, "sample_created_in_genotype_db": "2016-09-15", "sex": null, "comment": null, "_id": "0011P003"}']
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0011P003.
Running command ['/home/proj/bin/conda/envs/S_vogue/bin/vogue', 'load', 'genotype', '-s', '{"status": null, "sample_created_in_genotype_db": "2016-09-15", "sex": null, "comment": null, "_id": "0011P004"}']
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:Added sample 0011P004.
...

**Review:**
- [x] code approved by @adrosenbaum 
- [x] tests executed by @mayabrandi on hasta. 23/1-2020
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is patch  **version bump** because its not adding anything new to cg.

### CODE REVIEW

**Comment by mayabrandi** [comment]
Loaded the genotype-stage database into the trending database on hasta. All samples were loaded properly.

**Comment by mayabrandi** [comment]
@adrosenbaum I have assigned this PR to you. Could you please help me with the code review. It would be great!

**Comment by adrosenbaum** [comment]
> @adrosenbaum I have assigned this PR to you. Could you please help me with the code review. It would be great!

Sure, I'll have a look at it!

File: cg/apps/vogue.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
```
Comment (adrosenbaum) [inline cg/apps/vogue.py]: Looks fine! If this is tested and works on stage I have nothing to remark on 👍

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**